vip-scanner/checks/VIPRestrictedCommandsCheck.php

```diff
@@ -106,6 +106,7 @@ class VIPRestrictedCommandsCheck extends CodeCheck
 			"show_admin_bar"            => array( "level" => "Blocker", "note" => "The WordPress.com admin bar cannot be removed as it’s integral to the user experience on WordPress.com" ),
 			"serialize"                 => array( "level" => "Blocker", "note" => "Serialized data has <a href='https://www.owasp.org/index.php/PHP_Object_Injection'>known vulnerability problems</a> with Object Injection. JSON is generally a better approach for serializing data." ),
 			"unserialize"               => array( "level" => "Blocker", "note" => "Serialized data has <a href='https://www.owasp.org/index.php/PHP_Object_Injection'>known vulnerability problems</a> with Object Injection. JSON is generally a better approach for serializing data." ),
+			"get_intermediate_image_sizes" => array( "level" => "Blocker", "note" => "Intermediate images do not exist on VIP." ),
```

Comment:
Also, more context for the issue is better and would prevent some curious questions in the tickets :)